themes/wds_headless/inc/acf-pro.php

```diff
@@ -67,6 +67,74 @@ function wds_acf_blocks_init() {
 				'supports'        => $supports,
 			]
 		);
+
+		// Banners.
+		acf_register_block_type(
+			[
+				'name'            => 'banners',
+				'title'           => esc_html__( 'Banner', 'wds' ),
+				'description'     => esc_html__( 'A block to display a banners post.', 'wds' ),
+				'render_callback' => '',
+				'category'        => 'wds-content',
+				'icon'            => 'images-alt2',
+				'keywords'        => [ 'banners' ],
+				'mode'            => 'edit',
+				'enqueue_assets'  => '',
+				'align'           => 'wide',
+				'supports'        => $supports,
+			]
+		);
+
+		// Explore block.
+		acf_register_block_type(
+			[
+				'name'            => 'explore-list',
+				'title'           => esc_html__( 'Explore List', 'wds' ),
```

Comment:
not sure if we need the 'wds' [here](https://developer.wordpress.org/reference/functions/esc_html__/)

Reply:
@KobbyMmo, that's just the theme name string. It is not a required parameter...but having `wds` is better than having `default` (a string that will be added if you don't add anything)